runtime-extension-model/src/main/java/org/mule/runtime/config/internal/validation/CoreValidationsProvider.java

```diff
@@ -47,6 +47,7 @@ public List<Validation> get() {
                                                           new NamedTopLevelElementsHaveName(),
                                                           new NameHasValidCharacters(),
                                                           new NameIsNotRepeated(),
+                                                          new AllComponentsBelongToSomeExtensionModel(),
```

Comment:
maybe this should run earlier, beofre other validations that rely on the data from the extension model?

Reply:
makes sense

Reply:
done